### ISSUE TITLE
We need to clean the wiki

### ISSUE THREAD

**Issue by Gustry** [inasafe/inasafe#3868]
### Problem

I think there is a lot of old wiki pages which are outdated : 
* How to write impact function : https://github.com/inasafe/inasafe/wiki/Writing-impact-functions
* Jenkins
* ...

### Proposed Solution
We need to do some cleanup there.

**Comment by timlinux** [comment]
please make a calendar invite and invite us to have a wiki cleanup session otherwise this ticket will become one of the 200 orphans....

**Comment by Gustry** [comment]
Yes I agree. But this is a very low priority. We will do it after 4.0
